h3/src/client.rs

```diff
@@ -209,17 +210,40 @@ where
     }
 }
 
-pub struct RequestStream<S> {
+pub struct RequestStream<T>(RequestStreamImpl<FrameStream<T>>)
+where
+    T: quic::RecvStream;
+
+impl<T> Deref for RequestStream<T>
```

Comment:
It's a strongly held belief in the Rust community that `Deref` should _only_ be used for smart pointers, not for extending an inner type.

Reply:
Thanks, removed.